### ISSUE TITLE
Code blocks in PyMDown tabs

### ISSUE THREAD

**Issue by pawamoy** [KyleKing/mdformat-mkdocs#23]
The following markup:

````md
=== "duty"
    ```python title="duties.py"
    @duty(silent=True)
    def coverage(ctx):
        ctx.run("coverage combine", nofail=True)
        ctx.run("coverage report --rcfile=config/coverage.ini", capture=False)
        ctx.run("coverage html --rcfile=config/coverage.ini")


    @duty
    def test(ctx, match: str = ""):
        py_version = f"{sys.version_info.major}{sys.version_info.minor}"
        os.environ["COVERAGE_FILE"] = f".coverage.{py_version}"
        ctx.run(
            ["pytest", "-c", "config/pytest.ini", "-n", "auto", "-k", match, "tests"],
            title="Running tests",
        )
    ```
````

...gets formatted to:

`````md
=== "duty"
    ```python title="duties.py"
    @duty(silent=True)
    def coverage(ctx):
        ctx.run("coverage combine", nofail=True)
        ctx.run("coverage report --rcfile=config/coverage.ini", capture=False)
        ctx.run("coverage html --rcfile=config/coverage.ini")
    ```

````
@duty
def test(ctx, match: str = ""):
    py_version = f"{sys.version_info.major}{sys.version_info.minor}"
    os.environ["COVERAGE_FILE"] = f".coverage.{py_version}"
    ctx.run(
        ["pytest", "-c", "config/pytest.ini", "-n", "auto", "-k", match, "tests"],
        title="Running tests",
    )
```
````
`````

...while it actually shoudn't be formatted at all :slightly_smiling_face: (already correct).

```console
% mdformat --version
mdformat 0.7.17 (mdformat_tables: 0.4.1, mdformat_wikilink: 0.2.0, mdformat_admon: 2.0.2, mdformat_gfm: 0.3.6, mdformat_footnote: 0.1.1, mdformat_frontmatter: 2.0.8, mdformat_simple_breaks:
0.0.1, mdformat_mkdocs: 2.0.8, mdformat_config: 0.1.3, mdformat_ruff: 0.1.3, mdformat_web: 0.1.0, mdformat_beautysh: 0.1.1)
```

(Hi @KyleKing, amazing project! :rocket:)

**Comment by KyleKing** [comment]
Hi @pawamoy! Thanks for trying these plugins out

I can replicate the issue and I think the problem might be with `mdformat-admon` marking the end of a code block after two blank lines. If it doesn't take too long, I should have a patch ready this afternoon!

**Comment by pawamoy** [comment]
Wow, so fast, thanks a lot! :smile:

**Comment by KyleKing** [comment]
Thanks, but unfortunately it turned out to be a busier evening than expected. I found where I think the issue is, so I should have a patch this week at the latest!

**Comment by pawamoy** [comment]
Of course, no rush, thanks a lot for your help!

**Comment by KyleKing** [comment]
Fixed and should be available shortly in `mdformat-mkdocs v2.0.9` (https://github.com/KyleKing/mdformat-admon/commit/01c47f4ce0ef228e77685c93565a831ffad70aaf)!

**Comment by pawamoy** [comment]
Fantastic, thanks, will do!
